source/features/link-to-github-io.tsx

```diff
@@ -10,8 +10,6 @@ function getLinkToGitHubIo(repoTitle: HTMLElement, className?: string): JSX.Elem
 	return (
 		<a
 			href={`https://${repoTitle.textContent!.trim()}`}
-			target="_blank"
-			rel="noopener noreferrer"
 			className={className}
 		>
 			<LinkExternalIcon className="v-align-middle"/>
```

Comment:
@yakov116 thoughts on this? Should we change it to https://primer.style/octicons/globe-16 or https://primer.style/octicons/link-16?

Reply:
> [primer.style/octicons/globe-16](https://primer.style/octicons/globe-16)